apps/api/v1/rest/websoc/package.json

```
@@ -9,11 +9,13 @@
   "dependencies": {
     "@aws-sdk/client-lambda": "3.388.0",
     "@libs/db": "workspace:*",
+    "@libs/websoc-utils": "workspace:*",
```

Not a big deal, and maybe we can do this during a different "workspace cleanup PR", but I think I'm fine with the default `workspace:^` version used by pnpm. I'm not even sure if the versions mean anything because it's found locally, but maybe using the default is better so that way `pnpm install <package from monorepo>` will have the correct workspace version (I'm pretty sure `workspace:*` should only exist if you manually type that into the `package.json`)

```suggestion
    "@libs/websoc-utils": "workspace:^",
```

---

Tracked in #89.